src/components/Menu/index.js

```diff
@@ -23,37 +23,25 @@ function Menu() {
           <Link to="/visits">All Visits</Link>
         </li>
         <li>
-          <Link to="/visits/details/test-id">View a single Visit</Link>
+          <Link to="/visits/details-id">View a single Visit</Link>
         </li>
         <li>
           <Link to="/visits/create-new">New visit</Link>
         </li>
         <li>
-          <Link to="/visits/edit/test-id">Edit a visit</Link>
-        </li>
-        <li>
-          <Link to="/visits">All Visits</Link>
-        </li>
-        <li>
-          <Link to="/visits/details/test-id">View a single Visit</Link>
-        </li>
-        <li>
-          <Link to="/visits/create-new">New visit</Link>
-        </li>
-        <li>
-          <Link to="/visits/edit/test-id">Edit a visit</Link>
+          <Link to="/visits/edit-id">Edit a visit</Link>
         </li>
         <li>
           <Link to="/patients">Patients</Link>
         </li>
         <li>
-          <Link to="/patients/details/test-id">Patient Details</Link>
+          <Link to="/patients/details-id">Patient Details</Link>
         </li>
         <li>
           <Link to="/patients/create-new">Patient Create New</Link>
         </li>
         <li>
-          <Link to="/patients/edit/test-id">Patient Edit</Link>
+          <Link to="/patients/edit-id">Patient Edit</Link>
```

Comment:
The config path is "/patients/edit/:id".
So the correct link here is "/patients/edit/edit-id"